apps/site/pages/components/molecules/carousel.mdx

```diff
@@ -280,17 +340,107 @@ For further customization, you can use the following data attributes:
 
 ---
 
+## Examples
+
+### Using `ProductCard` component
+
+For the mobile preview, please refresh the browser after resize it.
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+  `itemsPerPage = 3`
+    <OverviewSection style={{ width: '100%', overflow: 'hidden' }}><CarouselUsage /></OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+    const isMobile = window.innerWidth <= 768
+
+    <Carousel
+      itemsPerPage={isMobile ? 1 : 3}
+      variant="scroll"
+      infiniteMode={false}
+    >
+      {products.map(({ product }, idx) => (
+        <ProductCard key={idx}>
+          <ProductCardImage>
+            <img
+              data-fs-image
+              src={product.image[0].url}
+              alt={product.image[0].alternateName}
+            />
+          </ProductCardImage>
+          <ProductCardContent
+            title={product.isVariantOf.name}
+            price={{
+              value: product.offers.offers[0].price,
+              listPrice: product.offers.offers[0].listPrice,
+              formatter: useFormattedPrice,
+            }}
+          />
+        </ProductCard>
+      ))}
+    </Carousel>
+
+    ```
+
+  </Tab>
+</Tabs>
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
+  <Tab>
+  `itemsPerPage = 8`
+    <OverviewSection style={{ width: '100%', overflow: 'hidden' }}><CarouselUsage itemsPerPage={8}/></OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+    const isMobile = window.innerWidth <= 768
+
+    <Carousel
+      itemsPerPage={isMobile ? 1 : 8}
+      variant="scroll"
+      infiniteMode={false}
+    >
+      {products.map(({ product }, idx) => (
+        <ProductCard key={idx}>
+          <ProductCardImage>
+            <img
+              data-fs-image
+              src={product.image[0].url}
+              alt={product.image[0].alternateName}
+            />
+          </ProductCardImage>
+          <ProductCardContent
+            title={product.isVariantOf.name}
+            price={{
+              value: product.offers.offers[0].price,
+              listPrice: product.offers.offers[0].listPrice,
+              formatter: useFormattedPrice,
+            }}
+          />
+        </ProductCard>
+      ))}
+    </Carousel>
+    ```
+  </Tab>
+</Tabs>
+
+<Callout type="warning" emoji="⚠️">
+Considering the component structure placed inside the `Carousel`, determine the ideal `itemsPerPage` number to accommodate the items and achieve the desired UI properly.
+</Callout>
+
+---
+
 ## Best Practices
 
 ### ✅ Do's
 
 - Always use more than 1 item per page so then the component keeps the proposed navigation behavior for both versions (mobile and desktop).
```

Comment:
I don't think this is always true, the Hero section could be an example of having 1 item per page on both mobile and desktop working well.

Reply:
agree! I'll remove it and replace it with `Choose an itemsPerPage value that effectively accommodate the items within the slide without compromising the UI. (Consider proposing different values for mobile and desktop)`